src/components/tables/table-children/TableChildBody.tsx

```diff
@@ -1,70 +1,90 @@
 import * as React from 'react';
 import * as classNames from 'classnames';
-import { ITableProps, IData, ITableHeadingAttribute } from '../Table';
+import * as _ from 'underscore';
+import { ITableHeadingAttribute, IData } from '../Table';
 import { getTableChildComponentId } from '../TableUtils';
 import { TableChildComponent, TOGGLE_ARROW_CELL_COUNT } from '../TableConstants';
 import { TableCollapsibleRowConnected } from '../TableCollapsibleRowConnected';
 import { TableRowWrapper } from '../TableRowWrapper';
 import { TableHeadingRowConnected } from '../TableHeadingRowConnected';
 import { convertUndefinedAndNullToEmptyString } from '../../../utils/FalsyValuesUtils';
 import { JSXRenderable } from '../../../utils/JSXUtils';
+import { IActionOptions } from '../../actions/Action';
 
-export const TableChildBody = (props: ITableProps): JSX.Element[] => {
-  const tableData = props.tableCompositeState.data || props.initialTableData;
-  return tableData.displayedIds.map((id: string, yPosition: number): JSX.Element => {
-    const rowData: IData = tableData.byId[id];
-    const rowWrapperId = `${getTableChildComponentId(props.id, TableChildComponent.TABLE_ROW_WRAPPER)}${rowData.id}`;
-    const headingAndCollapsibleId = `${getTableChildComponentId(props.id, TableChildComponent.TABLE_HEADING_ROW)}${rowData.id}`;
+export interface IInheritedFromTableProps {
```

Comment:
Is this interface used by something else than the table? I'd expect the name to be ITableBodyInheritedFromTableProps or something similar if that's not the case

If it's used by something else (e.g., the table action bar) I'd expect it to be somewhere more generic